deploy/helm/kubecf/templates/database.yaml

```diff
@@ -387,4 +409,67 @@ spec:
           {{- end }}
           restartPolicy: Never
 
+---
+apiVersion: quarks.cloudfoundry.org/v1alpha1
+kind: QuarksJob
+metadata:
+  name: database-migrate-charset
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+spec:
+  trigger:
+    strategy: manual
```

Comment:
So you run this manually based on docs?

Reply:
Yep!

Reply:
I plan on putting this directly into the release notes.